### ISSUE TITLE
Check for update not accurate

### ISSUE THREAD

**Issue by onetrev** [ZeroX-DG/SnippetStore#53]
I'm running 0.2.11 and using the "Check for Updates" features tells me I'm on the latest version. However, 0.2.12 is now out so it should be alerting me to that right?

On a side note, it would be cool if there could be an option to have the check for update run automatically every so often and automatically alert you to a new version.

**Comment by ZeroX-DG** [comment]
Yes, this is definitely a bug. I'll try to fix this issue. Thanks for reporting!

**Comment by onetrev** [comment]
Note... the Snippet Store website still has a download link to v0.2.11. But I used the [GitHub link](**https://github.com/ZeroX-DG/SnippetStore/releases**) to install v0.2.12. 

However, when I go the About menu in SnippetStore it still says I'm running 0.2.11, even though 0.2.12 is definitely installed (Windows lists 0.2.12 in the programs listing). And now check for updates chucks some error messages, see attached. Not sure what is happening, but I hope this is helpful for debugging.

![Annotation 2019-06-01 154424](https://user-images.githubusercontent.com/597601/58744293-27375d00-8484-11e9-9389-affd04f82d09.jpg)

**Comment by cmarabate** [comment]
> Note... the Snippet Store website still has a download link to only v0.2.11. But I used the [GitHub link](https://github.com/ZeroX-DG/SnippetStore/releases) to install v0.2.12.
> 
> However, when I go the About menu in SnippetStore it still says I'm running 0.2.11, even though 0.2.12 is definitely installed (Windows lists 0.2.12 in the programs listing). And now check for updates chucks some error messages, see attached. Not sure what is happening, but I hope this is helpful for debugging.
> 
> ![Annotation 2019-06-01 154424](https://user-images.githubusercontent.com/597601/58744293-27375d00-8484-11e9-9389-affd04f82d09.jpg)

Same exact issue for me as well on Windows 10 PC.
